### PR TITLE
RavenDB-21454: Splitting `LicenseAttribute.ServerWideTasks` into `LicenseAttribute.ServerWideBackup` and `LicenseAttribute.ServerWideExternalReplication` properties

### PR DESCRIPTION
**Pull request by ArieSLV**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-21454

### Additional description

We encountered an issue with configuring server-wide backup on a free cloud instance.

This aspect forces us to split `LicenseAttribute.ServerWideTasks` into `LicenseAttribute.ServerWideBackup` and `LicenseAttribute.ServerWideExternalReplication` properties.

### Type of change

- Bug fix

### How risky is the change?

- Low 

### Backward compatibility

- Non breaking change

### Is it platform specific issue?

- No

### Documentation update

- This change requires a documentation update.

### Testing by Contributor

- It has been verified by manual testing

### Testing by RavenDB QA team

- No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?

- No

### UI work

- No UI work is needed

### CODE REVIEW

**Review by ppekrol — APPROVED** [review]

File: src/Raven.Server/ServerWide/ClusterStateMachine.License.cs [hunk 1]
```diff
@@ -499,9 +499,15 @@ private static void AssertServerWideFor(ServerStore serverStore, LicenseAttribut
...
-                    throw new LicenseLimitException(LimitType.ServerWideTasks, "Your license doesn't support adding server wide tasks.");
+            case LicenseAttribute.ServerWideBackups:
+                if (serverStore.LicenseManager.LicenseStatus.HasServerWideBackups == false)
+                    throw new LicenseLimitException(LimitType.ServerWideBackups, "Your license doesn't support adding server wide backups.");
+
+                break;
+
+            case LicenseAttribute.ServerWideExternalReplications:
+                if (serverStore.LicenseManager.LicenseStatus.HasServerWideExternalReplications == false)
+                    throw new LicenseLimitException(LimitType.ServerWideExternalReplication, "Your license doesn't support adding external replication.");
```
Comment (ppekrol) [inline src/Raven.Server/ServerWide/ClusterStateMachine.License.cs]: server wide external replication

**Comment by kalczur** [comment]
I'll fix the studio build

File: src/Raven.Server/ServerWide/ClusterStateMachine.License.cs (hunk 1 above)
Comment (ArieSLV) [reply src/Raven.Server/ServerWide/ClusterStateMachine.License.cs]: Done

File: src/Raven.Server/ServerWide/ClusterStateMachine.License.cs [hunk 2]
```diff
@@ -499,9 +499,15 @@ private static void AssertServerWideFor(ServerStore serverStore, LicenseAttribut
...
-                    throw new LicenseLimitException(LimitType.ServerWideTasks, "Your license doesn't support adding server wide tasks.");
+            case LicenseAttribute.ServerWideBackups:
+                if (serverStore.LicenseManager.LicenseStatus.HasServerWideBackups == false)
+                    throw new LicenseLimitException(LimitType.ServerWideBackups, "Your license doesn't support adding server wide backups.");
+
+                break;
+
+            case LicenseAttribute.ServerWideExternalReplications:
+                if (serverStore.LicenseManager.LicenseStatus.HasServerWideExternalReplications == false)
+                    throw new LicenseLimitException(LimitType.ServerWideExternalReplications, "Your license doesn't support adding external replication.");
```
Comment (ppekrol) [inline src/Raven.Server/ServerWide/ClusterStateMachine.License.cs]: server wide?

Comment (ArieSLV) [reply src/Raven.Server/ServerWide/ClusterStateMachine.License.cs]: Done

**Pull request merged**